services/minespace-web/src/constants/strings.js

```diff
@@ -9,8 +9,11 @@ export const RETURN_PAGE_TYPE = {
   LOGOUT: "logout",
 };
 
-export const EMPTY_FIELD = "-";
+export const EMPTY_FIELD = "N/A";
```

Comment:
CORE has more options.... do we want to change all the - to N/A, or add a new one and rename things. 

```export const EMPTY_FIELD = "N/A";
export const EMPTY = "";
export const ZERO = "0.00";
export const UNASSIGNED = "Unassigned";```

Reply:
Not sure. I thought I would update it to match what we're using in Core and I think with the new mockups the way to indicate an empty field might change. I can revert it if you want?

Reply:
Well, now that I think about it again, it doesn't make sense. And that makes me think that it doesn't make sense for where we use it in Core either. In most cases it IS an applicable field, we just don't have a value for it--that doesn't mean it's not applicable, just empty.